internal/tool/ssh/impl/babysitter.go

```diff
@@ -99,6 +99,7 @@ func RunBabysitter(ctx context.Context) error {
 		SameProcess:   b.dep.App.SameProcess,
 		Sections:      b.dep.App.Sections,
 		SingleProcess: b.dep.SingleProcess,
+		SingleMachine: false,
```

Comment:
this should be set by default to false, maybe no need to set it explicitly.

Reply:
Done.